pywb/rewrite/rewriteinputreq.py

```diff
@@ -5,6 +5,13 @@
 from six.moves.urllib.parse import urlsplit
 import re
 
+try: # pragma: no cover
+    import brotli
+    has_brotli = True
+except:  # pragma: no cover
```

Comment:
I think it would be good practice to start using named exceptions when possible.

Here it would be `except ImportError` in order to be clearer about why we expect an exception and to be pythonic :)

Reply:
In general, yes, but here want to catch any `Exception` as that means brotli is not properly installed. Added `Exception`